CRM/Report/Form.php

```diff
@@ -814,6 +814,11 @@ public function setDefaultValues($freeze = TRUE) {
                 $this->_defaults["{$fieldName}_relative"] = $field['default'];
               }
             }
+            elseif (CRM_Utils_Array::value('type', $field) & CRM_Utils_Type::T_INT && is_array($field['default'])) {
+              $this->_defaults["{$fieldName}_min"] = CRM_Utils_Array::value('min', $field['default']);
+              $this->_defaults["{$fieldName}_max"] = CRM_Utils_Array::value('max', $field['default']);
+              $this->_defaults["{$fieldName}_value"] = 0;
```

Comment:
Assigning 0 will ignore/override what's set in $field['default'], although it won't reflect in test-build. So I would prefer it 
$this->_defaults["{$fieldName}_value"] = $field['default'];